jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -54,10 +54,10 @@ def bin_to_b58check(inp, magicbyte=b'\x00'):
     return b58encode(inp_fmtd + checksum)
 
 def bytes_to_hex_string(b):
-    return b.encode('hex')
+    return binascii.hexlify(b).decode('ascii')
```

Comment:
Am I missing something, or isn't this identical to `safe_hexlify`? It seems to only get used on line 264-265:

```python
def sha256(string):
    return bytes_to_hex_string(bin_sha256(string))
```

So I guess just change that call to `safe_hexlify` and remove this?

(Yes I realise this is probably about the pre-PR code, but prob just fix it now if I'm right).